octo/examples/02_finetune_new_observation_action.py

```diff
@@ -109,8 +109,8 @@ def process_batch(batch):
         LowdimObsTokenizer,
         n_bins=256,
         bin_type="normal",
-        low=-2.0,
-        high=2.0,
+        low=-5.5,
+        high=5.5,
```

Comment:
Are we using this file anywhere? If not, I think it's better if we leave it unchanged. As I understand this file is for the ALOHA dataset